tokens/main.py

```diff
@@ -3,16 +3,21 @@
 
 import base64
 import json
+
 import kubernetes.client
 import kubernetes.config
 import yaml
 from google.cloud import secretmanager
+from pathlib import Path
```

Comment:
Linter says `tokens/main.py:11:0: C0411: standard import "from pathlib import Path" should be placed before "import kubernetes.client" (wrong-import-order)`, but GitHub doesn't let me fix it as a suggestion :(.